boards/others/canbardo/canbardo.dts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2024-2025 Henrik Brix Andersen <henrik@brixandersen.dk>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <mem.h>
+#include <atmel/same70x20b.dtsi>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+#include "canbardo-pinctrl.dtsi"
+
+/ {
+	model = "CANbardo board";
+	compatible = "canbardo", "atmel,same70n20b", "atmel,same70b";
+
+	chosen {
+		zephyr,console = &uart1;
+		zephyr,shell-uart = &uart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,canbus = &can0;
+	};
+
+	aliases {
+		led0 = &dfu_led;
+		sw0 = &sw;
+		mcuboot-led0 = &dfu_led;
+		mcuboot-button0 = &sw;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		dfu_led: dfu_led {
+			gpios = <&piod 9 GPIO_ACTIVE_LOW>;
+			label = "DFU LED";
+		};
+
+		can_0_ledr: can_0_ledr {
+			gpios = <&piod 25 GPIO_ACTIVE_LOW>;
+			label = "CAN0 LED ERR";
+		};
+
+		can_0_ledg: can_0_ledg {
+			gpios = <&piod 22 GPIO_ACTIVE_LOW>;
+			label = "CAN0 LED RDY";
+		};
+
+		can_0_ledy: can_0_ledy {
+			gpios = <&piod 24 GPIO_ACTIVE_LOW>;
+			label = "CAN0 LED ACT";
+		};
+
+		can_1_ledr: can_1_ledr {
+			gpios = <&piod 19 GPIO_ACTIVE_LOW>;
+			label = "CAN1 LED ERR";
+		};
+
+		can_1_ledg: can_1_ledg {
+			gpios = <&piod 17 GPIO_ACTIVE_LOW>;
+			label = "CAN1 LED RDY";
+		};
+
+		can_1_ledy: can_1_ledy {
+			gpios = <&piod 18 GPIO_ACTIVE_LOW>;
+			label = "CAN1 LED ACT";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+
+		sw: sw {
+			label = "SW";
+			gpios = <&piod 15 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	transceiver0: can-phy0 {
+		compatible = "microchip,mcp2558fd", "can-transceiver-gpio";
+		enable-gpios = <&piod 26 GPIO_ACTIVE_LOW>;
+		max-bitrate = <8000000>;
```

Comment:
may not be a bad idea to switch these to use `DT_FREQ_M`

Reply:
I have refrained from doing that in the past since it reads wrong. A bitrate is not a frequency as in cycles per second, but rather measured as bit per second.

Reply:
mmh ok... should there be some aliases for the macros? `DT_BITRATE_M`? the issue is obviously the number readability on the file though if you have the wrong one here you'd find out pretty quickly

Reply:
Could be an enhancement, sure.